new/LIT6350MEMiracle.xml

```diff
@@ -0,0 +1,95 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6350MEMiracle" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">First miracle of ʾEwosṭātewos: the saint's heavenly journey from Armenia</title>
```

Comment:
Maybe:  Miracle of ʾEwosṭātewos: ...?
or is there a specific reason to write "first"?

Reply:
It would have been maybe a bit easier to manage the files later to assign the id with a keyword: LIT6350MEArmenia. I know it doesn't matter how the records are named but if you are looking later in oxygen it is easier to find specific miarcles. Just as a suggestion for later works. :)